src/services/user.service.js

```diff
@@ -4,4 +4,8 @@ export class UserService {
   static async register(user) {
     return await User.create(user);
   }
+
+  static async updatePassword(password, id) {
+    return await User.update({ password }, { where: { id: id } });
+  }
```

Comment:
This service should be reusable for updating user info. Instead of passing only a password, you can allow passing multiple data.